doc-content/enterprise-only/project-data/multiple-branches-project-con.adoc

```diff
@@ -0,0 +1,7 @@
+[id='multiple-branches-con']
+
+= Multiple branches in {CENTRAL}
+
+Multiple branch support in {CENTRAL} provides the ability to copy a project, including all of its assets, to a new branch. All new, imported, and sample projects open in the default *master* branch. You can create as many branches as you need and can work on multiple branches interchangeably without impacting the original project on the *master* branch.
```

Comment:
Here as well (just so that we don't forget about this part) - this also says 'copy a project'

Reply:
Reworded.